tests/test_mturk.py

```diff
@@ -30,132 +22,230 @@ class FixtureConfigurationError(Exception):
     """
 
 
+def system_marker():
+    # To prevent tests run on different systems trampling on each other,
+    # we mark data created in the MTurk sandbox with a value specific to
+    # each system.
+    return ':'.join(os.uname()).replace(' ', '')
```

Comment:
Not sure I want to leak info about my system this way; can we use `uuid.uuid4()` or does it need to persist across test runs?

Reply:
It's preferable if it persists, because if a tester force-quits the run before cleanup happens, there's still a chance for the cleanup to happen on the next run. I could md5 it or something... that would be just as good.